src/web/routes/application/middleware/set-previous-page.js

```diff
@@ -12,13 +12,13 @@ const getPreviousPage = (steps, step) => {
   }
 }
 
-const configureGet = (steps, step) => (req, res, next) => {
+const setPreviousPage = (steps, step) => (req, res, next) => {
   res.locals.previous = getPreviousPage(steps, step)
```

Comment:
Is it worth just calling this inside `configureGet` rather than creating a new piece of middleware for setting this single value?

Reply:
Moved setPreviousPage into configureGet/Post